Doesn't work as dev dependency

Hi, I'm trying to add this into a build as a development dependency so as not to depend on things outside the project. I get an error after doing this:

- `npm i -D singel`
- package.json script: `"lint": "eslint --ignore-path .gitignore . && singel src/singles/**/*.js",`
- `npm run lint`

My `/src/singles/Element.js` component:

```jsx
import React from "react";
const Element = props => <div {...props} />;
export default Element;
```

Output:
```
> npm run lint

> app@1.0.0 lint X:\app
> eslint --ignore-path .gitignore . && singel src/singles/**/*.js


X:\app\node_modules\babel-core\lib\transformation\file\index.js:558
      throw err;
      ^

SyntaxError: X:/hw-bb/ddhughes/recipe-app/src/singles/Element.js: Unexpected token (3:25)
  1 | import React from "react";
  2 |
> 3 | const Element = props => <div {...props} />;
    |                          ^
  4 |
  5 | export default Element;
  6 |
    at Parser.pp$5.raise (X:\hw-bb\ddhughes\recipe-app\node_modules\babylon\lib\index.js:4454:13)
    at Parser.pp.unexpected (X:\hw-bb\ddhughes\recipe-app\node_modules\babylon\lib\index.js:1761:8)
    at Parser.pp$3.parseExprAtom (X:\hw-bb\ddhughes\recipe-app\node_modules\babylon\lib\index.js:3750:12)
    at Parser.pp$3.parseExprSubscripts (X:\hw-bb\ddhughes\recipe-app\node_modules\babylon\lib\index.js:3494:19)
    at Parser.pp$3.parseMaybeUnary (X:\hw-bb\ddhughes\recipe-app\node_modules\babylon\lib\index.js:3474:19)
    at Parser.pp$3.parseExprOps (X:\hw-bb\ddhughes\recipe-app\node_modules\babylon\lib\index.js:3404:19)
    at Parser.pp$3.parseMaybeConditional (X:\hw-bb\ddhughes\recipe-app\node_modules\babylon\lib\index.js:3381:19)
    at Parser.pp$3.parseMaybeAssign (X:\hw-bb\ddhughes\recipe-app\node_modules\babylon\lib\index.js:3344:19)
    at Parser.pp$3.parseFunctionBody (X:\hw-bb\ddhughes\recipe-app\node_modules\babylon\lib\index.js:4226:22)
    at Parser.pp$3.parseArrowExpression (X:\hw-bb\ddhughes\recipe-app\node_modules\babylon\lib\index.js:4190:8)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! recipe-app@1.0.0 lint: `eslint --ignore-path .gitignore . && singel src/singles/**/*.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the app@1.0.0 lint script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\dh\AppData\Roaming\npm-cache\_logs\2018-07-03T10_37_38_718Z-debug.log
```

--- a/package.json
+++ b/package.json
@@ -50,6 +50,7 @@
   "dependencies": {
     "babel-plugin-transform-es2015-modules-commonjs": "^6.26.2",
     "babel-plugin-transform-require-stub": "^0.3.0",
+    "babel-preset-react-app": "^3.1.2",
     "babel-register": "^6.26.0",
     "chalk": "^2.4.1",
     "commander": "^2.16.0",

--- a/src/babelConfig.js
+++ b/src/babelConfig.js
@@ -1,5 +1,5 @@
-import transformES2015ModulesCommonJS from "babel-plugin-transform-es2015-modules-commonjs";
-import transformRequireStub from "babel-plugin-transform-require-stub";
+process.env.NODE_ENV = "development";
+process.env.BABEL_ENV = "development";
 
 const extensions = [
   "css",
@@ -15,8 +15,9 @@ const extensions = [
 ];
 
 export default {
+  presets: [require.resolve("babel-preset-react-app")],
   plugins: [
-    transformES2015ModulesCommonJS,
-    [transformRequireStub, { extensions }]
+    require.resolve("babel-plugin-transform-es2015-modules-commonjs"),
+    [require.resolve("babel-plugin-transform-require-stub"), { extensions }]
   ]
 };

